tests/copyFiles.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable max-len */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { faker } = require('@faker-js/faker');
+
+const { exec } = require('child_process');
+const util = require('util');
+const execAsync = util.promisify(exec);
+
+describe('File Copy', () => {
+  const baseCommand = 'node src/app.js';
+  let sourceContent = '';
+  const tempDir = path.join('tests', faker.word.noun());
+
+  const sourceFile = path.join(tempDir, faker.system.commonFileName('txt'));
+  const destinationFile = path.join(
+    tempDir,
+    faker.system.commonFileName('txt'),
+  );
+
+  beforeAll(() => {
+    fs.mkdirSync(tempDir);
+    sourceContent = faker.lorem.paragraphs();
+    fs.writeFileSync(sourceFile, sourceContent);
+  });
+
+  afterAll(() => {
+    fs.rmdirSync(tempDir, { recursive: true });
+  });
+
+  test('should throw an error if only one argument is provided', async() => {
```

Comment:
Students will start from success story
So it is more logical to test success flow first